code/style.css

```diff
@@ -1,4 +1,308 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: rgb(204, 204, 204);
+  color: rgb(0, 0, 0);
+  font-weight: normal;
+}
+
+body{
+  margin: 0px;
+  font-family: 'Raleway', sans-serif;
+}
+
+/*Header and navbar*/
+
+.flex-container {
+  height:75px;
+  background-color:darkgrey;
+  color:black;
+  font-size: 25px;
+  display:flex;
+  align-items: center;
+  justify-content: space-evenly;
+  flex:wrap;
+}
+
+.logo{
+  height:150px;
+}
+
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: inline;
+}
+
+a{
+  text-decoration: none;
+  font-size: 30px;
+  font-weight: bold;
+  padding:15px;
+  color:black;
+}
+
+a:hover {
+  opacity: 0.5;
+}
+
+
+/*Big news*/
+
+.big-news {
+  display: flex;
+  flex-direction: row;
+  margin:20px;
+  background-color: rgb(204, 204, 204);
+}
+
+
+.column_1 {
+  flex: 50%;
+  padding: 10px;  
+}
+
+.column_2{
+  flex: 25%;
+  padding: 10px;
+  margin:100px 300px 0px 0px;
+}
+
+.main-photo{
+  max-width:100%;
+  border-radius: 20px; 
+  box-shadow: 10px -16px darkgray;
```

Comment:
Regarding the requirements: you went over the blue level with your box shadow, bravo!